app/assets/stylesheets/forever_style_guide/modules/_hero.scss

```diff
@@ -70,7 +74,7 @@ TODO: use picture element + picturefill to avoid css hacks
   background-repeat: no-repeat;
 
   @media (min-width: $screen-xs) {
-    padding-top: 34%; /* (484 / 1400 * 100) image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
+    padding-top: percentage($hero-img-height / $hero-img-width); /* image sizes: 768px x 266px (xs) and 1400px x 484px (sm) */
```

Comment:
the percentage and variable approach should be used above here (line 72) as well, and then the comments on those particular lines would cease to be necessary.